example/mandelbrot.cpp

```diff
@@ -37,7 +36,7 @@ struct mandelbrot_fn
     result_type operator()(const point_t& p) const {
         // normalize the coords to (-2..1, -1.5..1.5)
         // (actually make y -1.0..2 so it is asymmetric, so we can verify some view factory methods)
-        double t=get_num_iter(point2<double>(p.x/(double)_img_size.x*3-2, p.y/(double)_img_size.y*3-1.0f));//1.5f));
+        double t=get_num_iter(point<double>(p.x/(double)_img_size.x*3-2, p.y/(double)_img_size.y*3-1.0f));//1.5f));
```

Comment:
Any idea why we are not using the previously defined `point_t` here ? (And similarly further down in the same struct)

Reply:
`point_t` is `point<ptrdiff_t>` not for double. 

BTW, There may be still numerous places left where the latter is used, waiting to  be replaced with the former.